build/media/libraries/redcore/media/redcore/css/component.bs3.css

```diff
@@ -9766,3 +9766,9 @@
 .redcore .redcore .modal-body {
   overflow: auto;
 }
+.redcore .translation-status {
```

Comment:
you should put this into less files too.... for both component.bs3 and component